tests/test_client.py

```diff
@@ -589,19 +589,21 @@ def test_profile(self):
             }
         }
 
-        with tempdir() as tmpdir:
-            with open(os.path.join(tmpdir, 'credentials.json'), 'w') as f:
-                f.write(json.dumps(creds))
+        for ext in ('yml', 'yaml', 'json'):
+            with tempdir() as tmpdir:
+                filename = 'credentials.{}'.format(ext)
+                with open(os.path.join(tmpdir, filename), 'w') as f:
+                    f.write(json.dumps(creds))
```

Comment:
Does this work with yaml?
I know there is a separate yaml python library that has a similar API.

Reply:
I validated that PyYAML can read json with `safe_load` so if he writes either yaml or json, the client lib can read it either way.

Reply:
YAML is a superset of JSON and fully supported by PyYaml

https://yaml.org/spec/1.2/spec.html#id2759572

Reply:
:facepalm: i didn't even catch that .... mis understood your comment.   change forth coming